devU-api/scripts/populate-db.ts

```diff
@@ -0,0 +1,170 @@
+    }
+}
+
+/*
+    -- Billy --
+    Student in 302
+
+    -- Bob --
+    Student in 312
+
+    -- Jones --
+    Student in 302 & 312, hasn't submitted anything
+*/
+async function RunRequests() {
+    try {
+        //Users
+        const userBilly = await SendPOST("/users", JSON.stringify({
+            email: "billy@buffalo.edu", externalId: "billy", preferredName: "Billiam"
+        }))
+        const userBob = await SendPOST("/users", JSON.stringify({
+            email: "bob@buffalo.edu", externalId: "bob", preferredName: "Bobby"
+        }))
+        const userJones = await SendPOST("/users", JSON.stringify({
+            email: "jones@buffalo.edu", externalId: "jones", preferredName: "Jones"
+        }))
+
+
+        //Courses
+        const course312 = await SendPOST("/courses", JSON.stringify({
+            name: "Web Applications", semester: "s2024", number: "CSE312", startDate: "2024-01-24", endDate: "2024-05-10"
+        }))
+        const course302 = await SendPOST("/courses", JSON.stringify({
+            name: "Intro to Experiential Learning", semester: "s2024", number: "CSE302", startDate: "2024-01-24", endDate: "2024-05-07"
+        }))
+
+
+        //UserCourse
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userBilly, courseId: course302, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userBob, courseId: course312, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userJones, courseId: course302, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userJones, courseId: course312, level: "student", dropped: false
+        }))
+        
+
+        //Assignments
+        const assign312_1 = await SendPOST("/assignments", JSON.stringify({
+            courseId: course312, name: "Homework 1", startDate: "2024-02-05", dueDate: "2024-02-26", endDate: "2024-03-11", gradingType: "code", categoryName: "Homework",
+            description: "HTTP", maxFileSize: 10000000, maxSubmissions: 10, disableHandins: false
+        }))
+        const assign312_2 = await SendPOST("/assignments", JSON.stringify({
+            courseId: course312, name: "Homework 2", startDate: "2024-02-19", dueDate: "2024-03-11", endDate: "2024-04-01", gradingType: "code", categoryName: "Homework",
+            description: "Authentication", maxFileSize: 10000000, maxSubmissions: 10, disableHandins: false
+        }))
+        SendPOST("/assignments", JSON.stringify({
+            courseId: course312, name: "Homework 3", startDate: "2024-03-04", dueDate: "2024-04-01", endDate: "2024-04-15", gradingType: "code", categoryName: "Homework",
+            description: "Image Uploads", maxFileSize: 10000000, maxSubmissions: 10, disableHandins: false
+        }))
+        // const assign312_quiz = await SendPOST("/assignments", JSON.stringify({
```

Comment:
Don't leave comment code in your files. Either uncomment, or delete. The later code for unimplemented entities is fine